common/txmgr/txmgr.go

```diff
@@ -201,15 +201,15 @@ func (b *Txm[CHAIN_ID, HEAD, ADDR, TX_HASH, BLOCK_HASH, R, SEQ, FEE]) Start(ctx
 
 		b.logger.Info("Txm starting runLoop")
 		b.wg.Add(1)
-		go b.runLoop()
+		go b.runLoop(ctx)
```

Comment:
```suggestion
		go b.runLoop()
```

Reply:
I need a context for some of the EVM APIs that are called (`SubscribeToKeyChanges` and `EnabledAddressesForChain`). Is there another option than to initiate a context.Background() ?

Reply:
Since this is a background routine, it needs to use a context derived from the stop chan. It can do that internally.